atoma-auth/src/sui/mod.rs

```diff
@@ -225,6 +226,7 @@ impl Sui {
     /// Returns an error if no USDC wallet is found for the active address.
     #[instrument(level = "info", skip_all, fields(address = %self.wallet_ctx.active_address().unwrap()))]
     async fn find_usdc_token_wallet(&mut self, usdc_package: ObjectID) -> Result<ObjectID> {
+        // TODO: This should return multiple USDC coins and merge them if the maximum coin is not enough.
```

Comment:
It's not wallet, we are just looking for the coins, because usdc is utxo based. Ideally we merge n coins, and we do split on the last one. So we get the precise amount.

Reply:
Are we doing this already ?

Reply:
No, I haven't figure out how to do that in rust. We do it in the dashboard when sending the usdc to the proxy.

Reply:
Is it safe to leave the code without the merging logic ?

Reply:
it's safish :-) but we should add the logic, but I haven't figured that out in the sui rustsdk yet